Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -32,6 +32,42 @@
       - NukeOpsUplink
       - LoneOpsUplink
 
+- type: listing
+  id: UplinkAmmoPouch
+  icon: { sprite: /Textures/_RMC14/Objects/Clothing/Pouches/large_ammo_mag.rsi, state: icon }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)